src/main/java/persistence/entity/persister/EntityPersister.java

```diff
@@ -0,0 +1,53 @@
+package persistence.entity.persister;
+
+import jakarta.persistence.Id;
+import jdbc.JdbcTemplate;
+import persistence.entity.exception.OptionalForbiddenException;
+import persistence.entity.exception.UnableToChangeIdException;
+import persistence.sql.dml.querybuilder.DeleteQueryBuilder;
+import persistence.sql.dml.querybuilder.InsertQueryBuilder;
+import persistence.sql.dml.querybuilder.UpdateQueryBuilder;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+import static persistence.sql.ddl.clause.primkarykey.PrimaryKeyValue.getPrimaryKeyValue;
+
+public class EntityPersister {
+    private final JdbcTemplate jdbcTemplate;
+
+    public EntityPersister(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public <T> T update(T entity, Long id) {
+        new OptionalForbiddenException().validate(entity);
+        String query = new UpdateQueryBuilder(entity.getClass()).getQuery(entity, id);
+        jdbcTemplate.executeUpdate(query);
+        return entity;
+    }
```

Comment:
```suggestion
    public <T> T update(T entity, Long id) {
        if (entity instanceof Optional) {
            throw new OptionalForbiddenException();
        }
        String query = new UpdateQueryBuilder(entity.getClass()).getQuery(entity, id);
        jdbcTemplate.executeUpdate(query);
        return entity;
    }
```
이런식으로 validate 하는게 매번 `OptionalForbiddenException` 을 생성하지 않을 수 있지 않을까요? :)